README.md

```diff
@@ -61,9 +61,18 @@ The best way to install this extension is to follow the instructions below. Howe
   3. Enter `about:debugging#/runtime/this-firefox`  in the address bar 
   4. Click `Load Temporary Add-on...` and navigate to the path of the downloaded `firefox.xpi` and select it to install it
 
-<strong>Firefox will remove the extension when the browser is closed. This will be solved once we provide a means to download the extension from the firefox store</strong>
+<strong>Firefox will remove the extension when the browser is closed if the extension is not downloaded from the store.</strong>
 
+## Firefox Nightly / Fennec F-droid / Mull (Android)
 
+1. Go to settings
+2. Scroll to the bottom and select "About {browser name}"
+3. Tap the browser logo five times
+4. Go back to settings and in the Advanced section, select "Custom Add-on collection"
+5. Type 17451592 as the collection owner (user ID) and "Accomodations" as the collection name. The browser will close to apply the settings.
+6. Go to Add-ons/Add-ons manager to install the add-on. 
```

Comment:
I think adding more clarity of the steps here would be most helpful for any reader since it is implied that they have to reopen firefox nightly again > settings > add-ons/add-ons manager and find jiffy reader